package.json

```diff
@@ -96,6 +96,7 @@
     "luxon": "^1.25.0",
     "microbundle": "^0.13.3",
     "ms.macro": "^2.0.0",
+    "multicall-query": "^0.1.5",
```

Comment:
TODO using my personal version just to test but we need to publish @uniswap/redux-multicall and use that instead.